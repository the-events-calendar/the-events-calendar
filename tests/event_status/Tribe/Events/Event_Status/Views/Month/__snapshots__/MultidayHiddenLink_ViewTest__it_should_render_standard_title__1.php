<?php return '<div class="tribe-events-calendar-month__multiday-event-hidden-title tribe-common-h8">
	Single Event 1</div>
';
