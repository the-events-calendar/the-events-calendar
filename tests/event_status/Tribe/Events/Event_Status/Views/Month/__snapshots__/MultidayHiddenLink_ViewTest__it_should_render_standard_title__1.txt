<h3 class="tribe-events-calendar-month__multiday-event-hidden-title tribe-common-h8">
	Single Event 1</h3>
